serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/client/brreg/BrregClientImpl.java

```diff
@@ -19,19 +20,28 @@
 
     private URI uri;
 
+    /**
+     * Creates a client configured to connect to http://data.brreg.no/
+     */
     public BrregClientImpl() {
         uri = URI.create("http://data.brreg.no/");
```

Comment:
Kan gjøre det i den andre branchen hvor jeg oppretter configuration for prod og staging